topics/S3/etcd-migration/main.py

```diff
@@ -0,0 +1,25 @@
+from src.common import bcolors, global_vars
+from src.helpers import get_arguments, set_args
+import src.steps as steps
+
+
+def do_migrate():
+    if not global_vars.skip_checks:
+        steps.system_health_check()
+        steps.preliminary_check()
+    steps.enter_migration_mode()
+    steps.enter_drain_mode()
+    steps.migrate_data()
+    steps.validate_migration()
+    steps.drain_and_restarts_all_hosts()
+    steps.validate_kwas_is_up()
+    steps.exit_migration_mode()
```

Comment:
you're skipping the "Perform IOs to test policy mapping sync" step, which is fine, it can be done manually after the script finishes running. but it should be explicitly stated somewhere that the script doesn't handle this step and recommend running it manually or something

Reply:
I didn't think that this test is part of the script scope. but I'll add this comment to the readme.